context/app/static/js/components/home/ExternalLinks/ExternalLinks.tsx

```diff
@@ -0,0 +1,108 @@
+        linkHref="https://www.protocols.io/workspaces/human-biomolecular-atlas-program-hubmap-method-development/"
+        linkText="HuBMAP Protocols"
+        description="View the HuBMAP Consortium’s protocols in protocols.io."
+      />
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/google_scholar.png`} height={56} width={56} alt="Google Scholar Logo" />}
+        linkHref="https://scholar.google.com/citations?user=CtGSN80AAAAJ"
+        linkText="HuBMAP Publications"
+        description="Explore the list of related publications generated by members of the HuBMAP Consortium on their Google Scholar page."
+      />
+    </Paper>
+  );
+}
+
+function RelatedTools() {
+  return (
+    <Paper>
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/hra_icon.png`} height={56} width={56} alt="HuBMAP Logo" />}
+        linkHref="https://humanatlas.io/"
+        linkText="Human Reference Atlas"
+        description="Navigate a comprehensive, high-resolution, three-dimensional atlas of all the cells in the healthy human body."
+      />
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/azimuth.png`} height={56} width={56} alt="HuBMAP Logo" />}
+        linkHref="https://azimuth.hubmapconsortium.org/"
+        linkText="Azimuth"
+        description="Azimuth uses a reference dataset to process, analyze, and interpret single-cell RNA-seq or ATAC-seq experiments."
+      />
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/fusion_icon.png`} height={56} width={56} alt="HuBMAP Logo" />}
+        linkHref="http://fusion.hubmapconsortium.org/?utm_source=hubmap"
+        linkText="FUSION"
+        description="Functional Unit State Identification and Navigation with WSI."
+      />
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/antibody_validation_reports.png`} height={56} width={56} alt="HuBMAP Logo" />}
+        linkHref="https://avr.hubmapconsortium.org/"
+        linkText="Antibody Validation Reports"
+        description="Provide antibody details for multiplex imaging assays and capture data requested by journals for manuscript submission."
+      />
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/hubmap_consortium.png`} height={56} width={56} alt="HuBMAP Logo" />}
+        linkHref="https://ingest.hubmapconsortium.org/"
+        linkText="HuBMAP Data Submission"
+        description="Submit data through the HuBMAP Data Ingest Portal as the central resource for registering, managing and ingesting data generated by the consortium."
+      />
+    </Paper>
+  );
+}
+
+function LinkSectionContainer({ children, title }: PropsWithChildren<{ title: string }>) {
+  return (
+    <Stack direction="column" maxWidth={{ md: '50%' }}>
+      <SectionHeader variant="h4" component="h3">
+        {title}
+      </SectionHeader>
+      <Paper>{children}</Paper>
+    </Stack>
+  );
+}
+
+function ExternalLinks() {
+  return (
+    <Box
+      display="flex"
+      flexDirection={{
```

Comment:
I imagine you used `Box` instead of `Stack` as it allows for these easy media queries?